runtime/src/main/java/org/corfudb/runtime/view/workflows/AddNode.java

```diff
@@ -44,9 +46,12 @@ public AddNode(@NonNull String endpointToAdd, @NonNull CorfuRuntime runtime,
 
     @Override
     protected UUID sendRequest(@NonNull ManagementClient managementClient) throws TimeoutException {
```

Comment:
I think later on there should a cleanup patch on all the inherited children of sendRequest because others are not taking care of NetworkException, and the signature suggests it only throws TimeoutException.

Reply:
Sounds good.